example/tests/elementActions.test.js

```diff
@@ -330,5 +338,32 @@ describe("Element Actions", () => {
         expect(await resultElement.text()).toEqual(remotePath);
     });
 
+    //Skipping the Webkit for download tests until the issue is resolved https://github.com/microsoft/playwright/issues/5396
```

Comment:
Please rewrite comment in TODO style (ie. `//TODO: un-skip when..`)
and also add same comment above test in 356

Reply:
Done